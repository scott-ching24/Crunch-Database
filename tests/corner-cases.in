set a 1 3 b c
set a 2 3 
snapshot absolutely nothing
snapshot
get a 2 4 6 3
get a
set b
get b
listKeys 2 3 1 3 2 
listKeys
listEntries s r 3 2 e r 3 sf
listEntries
bye
