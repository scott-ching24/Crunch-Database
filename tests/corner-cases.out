> ok

> ok

> invalid command

> saved as snapshot 1

> invalid command

> [2 3]

> invalid command

> no such key

> invalid command

> invalid command

> invalid command

> invalid command

> bye
